python/commands/download

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python3
```

Comment:
Can we put this in python/predictor-initializer?

Reply:
put into model-initializer.. as we discussed that makes more sense for now